src/main/antora/modules/websvcs/pages/rest/rest003.adoc

```diff
@@ -8,13 +8,13 @@ This section demonstrates the steps that are needed to create, build, deploy, an
 This section explains how to use NetBeans IDE to create a RESTful web service using a Maven archetype.
 The archetype generates a skeleton for the application, and you simply need to implement the appropriate method.
 
-You can find a version of this application at `_jakartaee-examples_/tutorial/jaxrs/hello/`.
+You can find a version of this application at `_jakartaee-examples_/tutorial/rest/hello/`.
 
 ==== To Create a RESTful Web Service Using NetBeans IDE
 
 . Ensure you have installed the tutorial archetypes as described in xref:intro:usingexamples/usingexamples.adoc#_installing_the_tutorial_archetypes[Installing the Tutorial Archetypes].
 
-. In NetBeans IDE, create a simple web application using the `jaxrs-service-archetype` Maven archetype.
+. In NetBeans IDE, create a simple web application using the `rest-service-archetype` Maven archetype.
```

Comment:
The archetype name may still start with `jaxrs`. Can you check? If not, I can.

Reply:
Okay, let me check

Reply:
@Harsh4902 so was this updated?

Reply:
@kito99 Yes, I have changed 'rest' with 'jaxrs'.